### ISSUE TITLE
process remains open after closing Sumatra

### ISSUE THREAD

**Issue by SisypheanFool** [sumatrapdfreader/sumatrapdf#822]
This used to be an occasional occurrence but now it always happens on latest build 3.1.2 x64.]
Not a big issue since this is such a low overhead program, but I wanted to report my experience FYI.
I use Sumatra on both Win7 x64 and Win10 devices.

**Comment by kjk** [comment]
I can't reproduce that.

How are you closing Sumatra? (alt-f4? closing the window with 'x') ?

Can you post some evidence of that e.g. a screenshot of Task Manager that shows Sumatra process is running.

Does it always happen when you do the following:
* start Task Manager
* kill Sumatra process if it's running, just to make sure
* start sumatra
* verify Sumatra process shows up in Task Manager
* exit sumatra
* Sumatra process still shows up in Task Manager

For me, the above sequence doesn't show Sumatra process, on win 10.

In other words, I need a repeatable way to reproduce the issue.

**Comment by SisypheanFool** [comment]
Thanks for looking at this. I can only routinely recreate it on my 
Win7 desktop. I can kill the open PIDs for Sumatra and it will start a new set of PIDs until I either kill it or do my periodic system reboots.
It's hit-or-miss occurrences on my Win10 laptop.
If you haven't already closed this ticket, please feel free to do so since I can't give you anything to work with and I assume it;'s something corrupt in the OS files on my desktop.

Thanks again.
